examples/Migration/CampaignManagement/CreateCompleteCampaignGoogleAdsApiOnly.php

```diff
@@ -0,0 +1,499 @@
+use Google\Ads\GoogleAds\V1\Enums\AdGroupCriterionStatusEnum\AdGroupCriterionStatus;
+use Google\Ads\GoogleAds\V1\Enums\AdGroupStatusEnum\AdGroupStatus;
+use Google\Ads\GoogleAds\V1\Enums\AdGroupTypeEnum\AdGroupType;
+use Google\Ads\GoogleAds\V1\Enums\AdvertisingChannelTypeEnum\AdvertisingChannelType;
+use Google\Ads\GoogleAds\V1\Enums\BudgetDeliveryMethodEnum\BudgetDeliveryMethod;
+use Google\Ads\GoogleAds\V1\Enums\CampaignStatusEnum\CampaignStatus;
+use Google\Ads\GoogleAds\V1\Enums\KeywordMatchTypeEnum\KeywordMatchType;
+use Google\Ads\GoogleAds\V1\Resources\Ad;
+use Google\Ads\GoogleAds\V1\Resources\AdGroup;
+use Google\Ads\GoogleAds\V1\Resources\AdGroupCriterion;
+use Google\Ads\GoogleAds\V1\Resources\AdGroupAd;
+use Google\Ads\GoogleAds\V1\Resources\Campaign;
+use Google\Ads\GoogleAds\V1\Resources\Campaign\NetworkSettings;
+use Google\Ads\GoogleAds\V1\Resources\CampaignBudget;
+use Google\Ads\GoogleAds\V1\Services\AdGroupCriterionOperation;
+use Google\Ads\GoogleAds\V1\Services\AdGroupOperation;
+use Google\Ads\GoogleAds\V1\Services\AdGroupAdOperation;
+use Google\Ads\GoogleAds\V1\Services\CampaignBudgetOperation;
+use Google\Ads\GoogleAds\V1\Services\CampaignOperation;
+use Google\Protobuf\BoolValue;
+use Google\Protobuf\Int64Value;
+use Google\Protobuf\StringValue;
+
+/**
+ * This code example is the last in a series of code examples that shows how to create
+ * a Search campign using the AdWords API, and then migrate it to the Google Ads API one
+ * functionality at a time. See CreateCompleteCampaignAdwordsApiOnly.php through
+ * CreateCompleteCampaignBothApisPhase4.php for code examples in various stages of migration.
+ *
+ * This code example represents the final state, where all the functionality - create a
+ * campaign budget, a Search campaign, ad groups, keywords and expanded text ads have been
+ * migrated to using the Google Ads API. The AdWords API is not used.
+ */
+class CreateCompleteCampaignGoogleAdsApiOnly
+{
+
+    // Number of ads being added/updated in this code example.
+    const NUMBER_OF_ADS = 5;
+    // The list of keywords being added in this code example.
+    const KEYWORDS_TO_ADD = [
+        "mars cruise",
+        "space hotel"
+    ];
+
+    // The default page size for search queries
+    const PAGE_SIZE = 1000;
+
+    /**
+     * Runs the CreateCompleteCampaignGoogleAdsApiOnly example.
+     * @param GoogleAdsClient $googleAdsClient the Google Ads API client
+     * @param string $customerId the client customer ID without hyphens
+     */
+    public static function runExample(GoogleAdsClient $googleAdsClient, string $customerId)
+    {
+        $campaignBudget = self::createCampaignBudget($googleAdsClient, $customerId);
+        $campaign = self::createCampaign($googleAdsClient, $customerId, $campaignBudget);
+        $adGroup = self::createAdGroup($googleAdsClient, $customerId, $campaign);
+        $adGroupAds = self::createTextAds($googleAdsClient, $customerId, $adGroup);
```

Comment:
$adGroupads and $keywords are not used. Please remove.

Reply:
Done.